core/src/components/datetime/datetime.md.scss

```diff
@@ -51,15 +51,14 @@
   line-height: 36px;
 }
 
+:host .calendar-body .calendar-month {
+  @include padding(4px, null);
```

Comment:
Since `.calendar-month-grid` uses `height: 100%` adding padding to this container will cause the contents to truncate sooner: 

![Screen Shot 2022-11-29 at 1 46 20 PM](https://user-images.githubusercontent.com/13732623/204618827-19cb5f21-73d8-49ff-8a6e-1e4ae5afd55a.png)

However, I don't see a strong justification on why that container needs `height: 100%`. Removing it and applying the padding to the `.calendar-month-grid` doesn't appear to cause regressions. I can swap this out and see what diffs it creates.